src/isPalindrome.js

```diff
@@ -19,7 +19,10 @@
  * @return {boolean}
  */
 function isPalindrome(str) {
-  // write code here
+  const strNormalise = str.replace(/[^\w+]/gi, '').toLowerCase();
```

Comment:
ты реплейсишь по строке
```suggestion
  const replacedString = str.replace(/[^\w+]/gi, '').toLowerCase();
```